llvm/test/CodeGen/AArch64/arm64-jumptable.ll

```diff
@@ -18,7 +18,7 @@ bb3:
 bb4:
   br label %exit.sink.split
 exit.sink.split:
-  %.sink = phi i32 [ 5, %bb4 ], [ %b, %bb1 ], [ 3, %bb3 ], [ %a, %entry ]
+  %.sink = phi i32 [ 5, %bb4 ], [ %b, %bb1 ], [ 7, %bb3 ], [ %a, %entry ]
```

Comment:
Can you explain this change?

Reply:
Because `%bb3` happens to meet this optimization condition, changing it to another value can prevent this test case from being affected.
See:
https://github.com/llvm/llvm-project/blob/bdc0afc87181d4f7ab8aad2da6fa70a1204f0a84/llvm/test/CodeGen/AArch64/arm64-jumptable.ll#L10-L21